### PR TITLE
Give more screen time to GUI wrappers

### PR DESCRIPTION
**Pull request by calmh**
Adds SyncTrayzor, rearranges a little.

Normal browser:

screen shot 2017-01-02 at 13 26 32

Phone:

screen shot 2017-01-02 at 13 26 08

### CODE REVIEW

**Comment by AudriusButkevicius** [comment]
Not sure if this empasizes enough that if you use windows use synctrayzor, it's more like 'it exists'.

**Comment by calmh** [comment]
Suggestions?

**Comment by scienmind** [comment]
Maybe some kind of "badge"/"logo" for windows, like the playstore link for android will do (but which logo?)

Or maybe rephrase or add under "third party contributions":
something like "**For starters**".


Anyways, it should be something obvious, "in your face" (like the playstore link):
Something intuitive for non-tech users who won't read all the stuff.

They land there - and there's lots of text, lots of links, lots of windows links in particular
And the "normal user" goes:
"_what do I download?_" "_probably the core? I don't wan't some 3-rd party crap_" "_but what do I do with it now? where's the buttons? Ok, I'll just get dropbox instead..._"


(Either way, this PR is already much better than what there is now :wink: )

**Comment by calmh** [comment]
OK:

screen shot 2017-01-02 at 20 59 01

There is of course much more that could be done graphically. Please do it, if you dear reader can visualize it in front of you and do it. But this way it's at least the first section the eye comes to and it sounds maybe a little bit more inviting.

**Comment by calmh** [comment]
@st-review merge it, everyone please feel free to continue improving

**Comment by st-review** [comment]
:ok_hand: Merged as 9c4bf79ba9bbe3b218d9b5955bb8de86ae96baab. Thanks, @calmh!
